provider/agent.go

```diff
@@ -101,6 +101,12 @@ func agentResource() *schema.Resource {
 				Optional:    true,
 				Description: "A URL to a document with instructions for troubleshooting problems with the agent.",
 			},
+			"motd_file": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Optional:    true,
+				Description: "The path to a file within the workspace containing a message to display to users when they login via SSH. A typical value would be /etc/motd.",
```

Comment:
Should we default to /etc/motd?

Reply:
It's a good question. I was thinking about the following:
- The path will be nonsensical on Windows, so the default would apply to Unix only
- Is it weird that it's on-by-default?

For instance, with on-by-default. Users using `alpine` as their workspace base would be greeted by:

```
Welcome to Alpine!

The Alpine Wiki contains a large amount of how-to guides and general
information about administrating Alpine systems.
See <http://wiki.alpinelinux.org/>.

You can setup the system with the command: setup-alpine

You may change this message by editing /etc/motd.
```

It's not necessarily a bad thing though, I'm definitely open to defaulting to `/etc/motd`.

Reply:
Let's leave it out for now, and if we see fit we can always add a default!